### ISSUE TITLE
WebStorm 2023.1 support

### ISSUE THREAD

**Issue by Grawl** [denofevil/import-cost#51]

**Comment by mohamedhk2** [comment]
Not compatible with the version of your running IDE (PhpStorm 2023.1)

**Comment by ezalivadnyi** [comment]
Any updates? I can't find "import cost" in WebStorm Plugins...

**Comment by Grawl** [comment]
WebStorm 2023.1.1 released

**Comment by Grawl** [comment]
@ezalivadnyi you can't find it because it's not supported your IDE version

here it is https://plugins.jetbrains.com/plugin/9970-import-cost

**Comment by austinbiggs** [comment]
*bump :)

**Comment by prn-feip** [comment]
WebStorm 2023.1.2 released

**Comment by Grawl** [comment]
WebStorm 2023.1.2 released

**Comment by anstarovoyt** [comment]
Hello! The problem is that in 22.3 there were a lot of complaints about the performance of the plugin, so I've decided to postpone updating until I have time to investigate the problem. Unfortunately, I didn't find the cause of high CPU usages, but I've reduced the number of threads for calculation, I hope that will help. 

Since 23.2 is almost out I think we can skip 23.1 and update the plugin directly to 23.2.
